torch/lib/THPP/tensors/generic/THTensor.cpp

```diff
@@ -113,7 +122,7 @@ auto THTensor<real>::resize(THLongStorage *size,
 
 template<>
 auto THTensor<real>::resizeAs(const Tensor& src) -> THTensor& {
-  THTensor_(resizeAs)(tensor, dynamic_cast<const THTensor<real>&>(src).tensor);
```

Comment:
Aren't members of `const` objects also automatically `const` (unless `mutable`)? Also, you can't "copy" a const pointer to a non const pointer, you'd need a type cast there. Keep in mind that `const Tensor*` means that `Tensor` can't be changed, not that the pointer can't be changed (`Tensor const*`).